bitnami/kafka/templates/scripts-configmap.yaml

```diff
@@ -1,39 +1,93 @@
-{{- if .Values.externalAccess.enabled }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "kafka.fullname" . }}-scripts
   labels: {{- include "kafka.labels" . | nindent 4 }}
 data:
+  {{- $fullname := include "kafka.fullname" . }}
+  {{- $releaseNamespace := .Release.Namespace }}
+  {{- if .Values.externalAccess.autoDiscovery.enabled }}
+  auto-discovery.sh: |-
+    #!/bin/bash
+
+    SVC_NAME="${MY_POD_NAME}-external"
+
+    {{- if eq .Values.externalAccess.service.type "LoadBalancer" }}
+    # Auxiliar functions
+    retry_while() {
+        local -r cmd="${1:?cmd is missing}"
+        local -r retries="${2:-12}"
+        local -r sleep_time="${3:-5}"
+        local return_value=1
+
+        read -r -a command <<< "$cmd"
+        for ((i = 1 ; i <= retries ; i+=1 )); do
```

Comment:
Is that option better in terms of performance? If there's no "real" reason to change this, I'll keep it as it's.

Reply:
Well, I am not sure about the performance, as `seq` require to call an external command, but in the other hand using the normal `for` it will perform a comparison and an incrementation in every iteration.
Anyway, I don't think this kind of performance is worrying here, I just suggest it because it is cleaner.

Reply:
I don't consider it a "cleaner" solution. I think it's very subjective statement. Anyway, I don't think it's something important.

Reply:
Ok, no problem, it was just my opinion but there is nothing wrong in have it as it is.